raiden/storage/migrations/v18_to_v19.py

```diff
@@ -69,7 +69,7 @@ def _add_blockhash_to_state_changes(storage: SQLiteStorage, cache: BlockHashCach
             data = json.loads(state_change.data)
             assert "block_hash" not in data, "v18 state changes cant contain blockhash"
             record = BlockQueryAndUpdateRecord(
-                block_number=int(data["block_number"]),
+                block_number=BlockNumber(int(data["block_number"])),
```

Comment:
Here, it is sufficient to do `BlockNumber(data["BlockNumber"])`

Reply:
Below are other instances as well.

Reply:
Are you sure? It doesn't seem to convert it:

```
In [3]: BlockNumber('1')
Out[3]: '1'
```

Reply:
No it doesn't convert. The type aliases are an identity function at runtime. 
We talked about it in person. The `int()` needs to stay in.